ltp-production.yaml

```diff
@@ -27,11 +31,442 @@ projects:
   - dragonboard-410c
   - x15
   - x86
+  - i386
   - qemu_x86_64
-  - qemu_x86_32
+  - qemu_i386
   - qemu_arm
   - qemu_arm64
   known_issues:
+  - environments:
+    - hi6220-hikey
+    notes: >
+      Linux-4.4: LTP: bind02: socket() failed: errno=EACCES(13): Permission
+      denied
+    projects:
+      - lkft/linaro-hikey-stable-rc-4.4-oe
+    test_name: ltp-syscalls-tests/bind02
+    url: https://bugs.linaro.org/show_bug.cgi?id=2962
+    active: true
+    intermittent: false
+
+  - environments:
+    - hi6220-hikey
+    - x15
+    - qemu_x86_64
+    - qemu_arm64
+    - qemu_arm
+    - qemu_i386
+    notes: >
+      LKFT: LTP: pselect01_64: slept for too long
+    projects: *projects_all
+    test_name: ltp-syscalls-tests/pselect01
+    url: https://bugs.linaro.org/show_bug.cgi?id=3089
+    active: true
+    intermittent: false
+  - environments:
+    - hi6220-hikey
+    - x15
+    - qemu_x86_64
+    - qemu_arm64
+    - qemu_arm
+    - qemu_i386
```

Comment:
I noticed this set of 4 qemu environments referenced a lot. Why not add a "- environments: &environments_qemu" global and use a reference instead of repeating it multiple times?